frontend/public/components/about-modal.tsx

```diff
@@ -74,6 +77,14 @@ const AboutModalItems: React.FC<AboutModalItemsProps> = ({ closeAboutModal }) =>
               </TextListItem>
             </>
           )}
+          {acmVersion && (
```

Comment:
We might need to make this part of another PR, or we'd need to hold this PR a few sprints. CONSOLE-2840 has dependencies on the ACM team and won't be finished soon.

Reply:
About modal changes moved to https://github.com/openshift/console/pull/9508.